### ISSUE TITLE
Add Merge functionality to Voyager

### ISSUE THREAD

**Issue by allanbolipata** [mskcc/beagle#250]
We need to add a feature that lets CAS merge projects that can be uploaded to portal.

Initial requirements:
1. From `labHeadName` or `labHeadEmail`, get all requests associated with it
2. Merge into cbioportal instance e-mail address without the @mskcc.org, prepending "set_" to it
    - Use either merge script from https://github.com/mskcc/roslin-internal-services/tree/master/cbioportal_merge or something similar
3. Figure out how to incorporate legacy merged projects
    - Use old MongoDB? Create new DB? Run old projects through Voyager again?
4. Have it post to JIRA

The merge script in no. 2 could work as-is, but it hasn't been touched for half a year and would need some review. Since I wrote it, I'm happy to look it over before we start this effort.

@songt please add any info as needed.

**Comment by allanbolipata** [comment]
Add redaction json to endpoint

@songt expand, what are redactions
